### DISCUSSION TITLE
Thank You for livewire

### DISCUSSION THREAD

**Discussion by agustinus-robert** [livewire/livewire#2734] [4. General]
Oke, i not want to disccuse anything, sorry for my bad english..

I just want to say, thank you for this framework, 2 years i confuse about JS, and now i am very happy because of this full stack framework livewire laravel.. Without touch JS again

My project sometime full of JS, at the point Jquery, it very difficult for me, as backend, i just want to use ajax, but sometime listener needed to add on my project..

And now, it gone, because of livewire.. Thank you friend

**Comment by calebporzio** [comment]
Your welcome!!!
